Add AppImage build instruction

@probonopd, could you add pullrequests with AppImage build script for latest AzPainter (currently 2.1.0)

> If the AzPainter source code was on GitHub, I would quickly have sent a Pull Request.
> https://github.com/AppImage/AppImages/issues/183#issuecomment-289288167

Also check if my repo still has this issues then, please, help fix it in `master` branch

REFERENCE:
* https://github.com/AppImage/AppImages/issues/183
* https://github.com/AppImage/AppImages/blob/master/recipes/meta/AzPainter.yml

Running into a build error:

```
rm -f libmlib.a

ar cru libmlib.a src/mAccelerator.o src/mBufOperate.o src/mColorConv.o src/mDef.o src/mFileList.o src/mFont_common.o src/mGui.o src/mGui_key.o src/mIOFileBuf.o src/mIniRead.o src/mIniWrite.o src/mList.o src/mMemAuto.o src/mMemDebug.o src/mMenu.o src/mMessageBox.o src/mNanoTime_common.o src/mRandXorShift.o src/mRectBox.o src/mStr.o src/mTextParam.o src/mTextRead.o src/mTree.o src/mUndo.o src/mUtil.o src/mUtilCharCode.o src/mUtilFile_common.o src/mUtilStdio.o src/mUtilStr.o src/image/mGIFDecode.o src/image/mGIFEncode.o src/image/mImageBuf.o src/image/mImageList.o src/image/mLoadImage.o src/image/mLoadImage_bmp.o src/image/mLoadImage_gif.o src/image/mLoadImage_jpeg.o src/image/mLoadImage_png.o src/image/mPSDLoad.o src/image/mPSDSave.o src/image/mPixbuf.o src/image/mPixbuf_2.o src/image/mSaveImage.o src/image/mSaveImage_bmp.o src/image/mSaveImage_gif.o src/image/mSaveImage_jpeg.o src/image/mSaveImage_png.o src/lib/mFTGSUB.o src/lib/mFontConfig.o src/lib/mFreeType.o src/lib/mZlib.o src/linux/mDirEntry.o src/linux/mEnumFont.o src/linux/mFile.o src/linux/mFont.o src/linux/mNanoTime.o src/linux/mPath.o src/linux/mThread.o src/linux/mUtilFile.o src/linux/mUtilSys.o src/private/mAppTimer.o src/private/mEditTextBuf.o src/private/mEventList.o src/private/mEvent_pv.o src/private/mLVItemMan.o src/private/mListViewArea.o src/private/mListViewPopup.o src/private/mMenuWindow.o src/private/mMenu_pv.o src/private/mMultiEditArea.o src/private/mTranslation.o src/private/mWidget_pv.o src/widget/mAboutDlg.o src/widget/mBitImgButton.o src/widget/mArrowButton.o src/widget/mButton.o src/widget/mCheckButton.o src/widget/mColorButton.o src/widget/mColorDialog.o src/widget/mColorPreview.o src/widget/mComboBox.o src/widget/mContainer.o src/widget/mContainerView.o src/widget/mContainer_layout.o src/widget/mDialog.o src/widget/mDockHeader.o src/widget/mDockWidget.o src/widget/mExpander.o src/widget/mFileDialog.o src/widget/mFileListView.o src/widget/mFontButton.o src/widget/mFontDialog.o src/widget/mGroupBox.o src/widget/mHSVPicker.o src/widget/mIconButtons.o src/widget/mInputAccelKey.o src/widget/mInputTextDlg.o src/widget/mLabel.o src/widget/mLineEdit.o src/widget/mListView.o src/widget/mMenuBar.o src/widget/mMultiEdit.o src/widget/mPopupProgress.o src/widget/mPopupWindow.o src/widget/mProgressBar.o src/widget/mScrollBar.o src/widget/mScrollView.o src/widget/mSliderBar.o src/widget/mSplitter.o src/widget/mTab.o src/widget/mTextButtons.o src/widget/mToolTip.o src/widget/mWidget.o src/widget/mWidgetBuilder.o src/widget/mWindow.o src/widget/mListHeader.o src/x11/x11_clipboard.o src/x11/x11_cursor.o src/x11/x11_dnd.o src/x11/x11_event.o src/x11/x11_event_sub.o src/x11/x11_im.o src/x11/x11_keyconv.o src/x11/x11_main.o src/x11/x11_pixbuf.o src/x11/x11_util.o src/x11/x11_widget.o src/x11/x11_window.o src/x11/x11_xinput2.o 
ar: src/image/mSaveImage_bmp.o: No such file or directory
make[2]: *** [libmlib.a] Error 1
make[2]: Leaving directory `/home/travis/build/probonopd/azpainter/mlib'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/travis/build/probonopd/azpainter'
make: *** [all] Error 2
```

Does anyone know how to fix this?
https://travis-ci.org/probonopd/azpainter/builds/270730518#L944

Here is what I did:
https://github.com/probonopd/azpainter/blob/patch-1/.travis.yml

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,7 +12,7 @@ install:
     - # source /opt/qt*/bin/qt*-env.sh
 
 script:
-  - ./configure --prefix=/usr
+  - bash ./configure --prefix=/usr
   - make -j$(nproc)
   - make install DESTDIR=$(readlink -f appdir) ; find appdir/
   - wget -c "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage"

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,0 +1,26 @@
+language: cpp
+compiler: gcc
+sudo: require
+dist: trusty
+
+before_install:
+    - sudo add-apt-repository ppa:beineri/opt-qt59-trusty -y
+    - sudo apt-get update -qq
+
+install:
+    - sudo apt-get -y install qt59base
+    - source /opt/qt*/bin/qt*-env.sh
+
+script:
+  - ./configure --prefix=/usr
+  - make -j$(nproc)
+  - make install DESTDIR=$(readlink -f appdir) ; find appdir/
+  - wget -c "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage"
+  - chmod a+x linuxdeployqt*.AppImage
+  - unset QTDIR; unset QT_PLUGIN_PATH ; unset LD_LIBRARY_PATH
+  - ./linuxdeployqt*.AppImage ./appdir/usr/share/applications/*.desktop -bundle-non-qt-libs
+  - ./linuxdeployqt*.AppImage ./appdir/usr/share/applications/*.desktop -appimage
+
+after_success:
+  - find ./appdir -executable -type f -exec ldd {} \; | grep " => /usr" | cut -d " " -f 2-3 | sort | uniq
+  - curl --upload-file ./APPNAME*.AppImage https://transfer.sh/APPNAME-git.$(git rev-parse --short HEAD)-x86_64.AppImage

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,12 +4,12 @@ sudo: require
 dist: trusty
 
 before_install:
-    - sudo add-apt-repository ppa:beineri/opt-qt59-trusty -y
-    - sudo apt-get update -qq
+    - # sudo add-apt-repository ppa:beineri/opt-qt59-trusty -y
+    - # sudo apt-get update -qq
 
 install:
-    - sudo apt-get -y install qt59base
-    - source /opt/qt*/bin/qt*-env.sh
+    - sudo apt-get -y install gcc make libx11-dev libxext-dev libxi-dev libfreetype6-dev libfontconfig1-dev zlib1g-dev libjpeg-dev libpng12-dev
+    - # source /opt/qt*/bin/qt*-env.sh
 
 script:
   - ./configure --prefix=/usr

--- a/.travis.yml
+++ b/.travis.yml
@@ -8,7 +8,7 @@ before_install:
     - # sudo apt-get update -qq
 
 install:
-    - sudo apt-get -y install aclocal gcc make libx11-dev libxext-dev libxi-dev libfreetype6-dev libfontconfig1-dev zlib1g-dev libjpeg-dev libpng12-dev
+    - sudo apt-get -y install automake gcc make libx11-dev libxext-dev libxi-dev libfreetype6-dev libfontconfig1-dev zlib1g-dev libjpeg-dev libpng12-dev
     - # source /opt/qt*/bin/qt*-env.sh
 
 script:

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,13 +3,8 @@ compiler: gcc
 sudo: require
 dist: trusty
 
-before_install:
-    - # sudo add-apt-repository ppa:beineri/opt-qt59-trusty -y
-    - # sudo apt-get update -qq
-
 install:
     - sudo apt-get -y install automake gcc make libx11-dev libxext-dev libxi-dev libfreetype6-dev libfontconfig1-dev zlib1g-dev libjpeg-dev libpng12-dev
-    - # source /opt/qt*/bin/qt*-env.sh
 
 script:
   - bash ./configure --prefix=/usr

--- a/.travis.yml
+++ b/.travis.yml
@@ -8,7 +8,7 @@ before_install:
     - # sudo apt-get update -qq
 
 install:
-    - sudo apt-get -y install gcc make libx11-dev libxext-dev libxi-dev libfreetype6-dev libfontconfig1-dev zlib1g-dev libjpeg-dev libpng12-dev
+    - sudo apt-get -y install aclocal gcc make libx11-dev libxext-dev libxi-dev libfreetype6-dev libfontconfig1-dev zlib1g-dev libjpeg-dev libpng12-dev
     - # source /opt/qt*/bin/qt*-env.sh
 
 script:

--- a/.travis.yml
+++ b/.travis.yml
@@ -23,4 +23,4 @@ script:
 
 after_success:
   - find ./appdir -executable -type f -exec ldd {} \; | grep " => /usr" | cut -d " " -f 2-3 | sort | uniq
-  - curl --upload-file ./APPNAME*.AppImage https://transfer.sh/APPNAME-git.$(git rev-parse --short HEAD)-x86_64.AppImage
+  - curl --upload-file ./AzPainter*.AppImage https://transfer.sh/AzPainter-git.$(git rev-parse --short HEAD)-x86_64.AppImage

